File: wrappers/java/src/main/java/org/hyperledger/indy/sdk/PoolConfigNotCreatedException.java
package org.hyperledger.indy.sdk;

public class PoolConfigNotCreatedException extends IndyException
{
	private final static String message = "The requested pool cannot be opened because it does not have an existing configuration.";

	PoolConfigNotCreatedException(int sdkErrorCode) 
    {
    	super(message, sdkErrorCode);
    }
}

File: wrappers/dotnet/indy-sdk-dotnet/PoolApi/PoolConfigNotCreatedException.cs
namespace Hyperledger.Indy.PoolApi
{
    /// <summary>
    /// Exception thrown when attempting to open pool which does not yet have a created configuration.
    /// </summary>
    public class PoolConfigNotCreatedException : IndyException
    {
        const string message = "The requested pool cannot be opened because it does not have an existing configuration.";

        /// <summary>
        /// Initializes a new PoolConfigNotCreatedException.
        /// </summary>
        internal PoolConfigNotCreatedException() : base(message, (int)ErrorCode.PoolLedgerNotCreatedError)
        {

        }
    }

}
